src/main/java/teammates/common/datatransfer/questions/FeedbackRankOptionsQuestionDetails.java

```diff
@@ -125,8 +142,8 @@ public String getQuestionWithExistingResponseSubmissionFormHtml(
 
         }
 
-        boolean isMinOptionsToBeRankedEnabled = minOptionsToBeRanked != Integer.MIN_VALUE;
-        boolean isMaxOptionsToBeRankedEnabled = maxOptionsToBeRanked != Integer.MIN_VALUE;
+        boolean isMinOptionsToBeRankedEnabled = minOptionsToBeRanked != NO_VALUE;
```

Comment:
So although not ideal to modify the state, unfortunately without a redesign of the class we can only set the instance variable, don't shadow it.